internal/controller/persistentvolume/persistentvolume.go

```diff
@@ -169,26 +169,16 @@ func newPVReconciler(mgr manager.Manager, config ctrl.Config) reconcile.Reconcil
 }
 
 func add(mgr manager.Manager, r reconcile.Reconciler) error {
-	// Create a new controller
-	c, err := controller.New(
-		"persistentvolume-controller",
-		mgr,
-		controller.Options{MaxConcurrentReconciles: 1, Reconciler: r})
-	if err != nil {
-		return err
-	}
+	// Create a new controller builder
+	build := builder.ControllerManagedBy(mgr).Named(
+		"persistentvolume-controller").WithOptions(
+		controller.Options{MaxConcurrentReconciles: 1})
```

Comment:
```suggestion
		controller.Options{MaxConcurrentReconciles: 1, Reconciler: r})
```
Reconciler not required here?

Reply:
its in complete below